esp/esdllib/esdl_script.cpp

```diff
@@ -109,8 +219,12 @@ class CEsdlScriptContext : public CInterfaceOf<IEsdlScriptContext>
     Owned<IModularTraceMsgSink>   jlogSink;
     Owned<IModularTraceMsgSink>   consoleSink;
     Owned<IDataMaskingEngine>     maskingEngine;
-    using MaskerStack = std::list<Owned<IDataMaskingProfileContext> >;
-    MaskerStack                   maskerStack;
+    using MaskerScope = Owned<IDataMaskingProfileContext>;
```

Comment:
A slightly confusing name - it isn't clear that it is an owned pointer, so I thought the code probably leaked.

Reply:
https://track.hpccsystems.com/browse/HPCC-29156